server/schedule/operator/create_operator.go

```diff
@@ -293,12 +293,21 @@ func CreateLeaveJointStateOperator(desc string, ci ClusterInformer, origin *core
 
 // CreateWitnessPeerOperator creates an operator that set a follower or learner peer with witness
 func CreateWitnessPeerOperator(desc string, ci ClusterInformer, region *core.RegionInfo, peer *metapb.Peer) (*Operator, error) {
-	brief := fmt.Sprintf("create witness: region %v peer %v on store %v", region.GetID(), peer.Id, peer.StoreId)
-	return NewOperator(desc, brief, region.GetID(), region.GetRegionEpoch(), OpRegion, region.GetApproximateSize(), BecomeWitness{StoreID: peer.StoreId, PeerID: peer.Id}), nil
+	brief := fmt.Sprintf("switch to witness: region %v peer %v on store %v", region.GetID(), peer.Id, peer.StoreId)
+	return NewOperator(desc, brief, region.GetID(), region.GetRegionEpoch(), OpAdmin, region.GetApproximateSize(), BecomeWitness{StoreID: peer.StoreId, PeerID: peer.Id}), nil
```

Comment:
Usually, the operator added through API is admin.

Reply:
Oh, I see, I'll change it to 0, consistent with Demote.